shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -329,14 +333,13 @@ static bool DidFlagChange(const flutter::SemanticsNode& oldNode,
     return object;
   }
 
-  SemanticsObject* candidate = nil;
```

Comment:
I am not sure what had got into my head when i write this code. The pr change it to be more readable without changing the functionality